vpplink/vpplink.go

```diff
@@ -53,12 +58,22 @@ func NewVppLink(socket string, logger *logrus.Entry) (*VppLink, error) {
 		return nil, errors.Wrap(err, "channel creation failed")
 	}
 
-	return &VppLink{
+	vppLink := &VppLink{
 		conn:   conn,
 		ch:     ch,
 		socket: socket,
 		log:    logger,
-	}, nil
+		pid:    uint32(os.Getpid()),
+	}
+
+	stopEvents, err := vppLink.watchInterfaceEvents()
```

Comment:
Maybe we want to expose this function to the user (a.k.a require running `vpp.WatchInterfaceEvents()` if we want enabling it). This with the hypothesis that `vpplink/` is shared with other projects

Reply:
Yeah, was thinking that too.. for the purpose of avoiding enabling it by default. However it is only needs to be enabled once so perhaps I can make it start automatically on the first call to [`WatchInterfaceEvents`](https://github.com/ondrej-fabry/vpp-dataplane/blob/link-status/vpplink/interfaces.go#L771) and stop it automatically when last watcher finishes.

Reply:
That way user does not need to do extra call.. which might actually return error if the user calls it multiple times.